From 83c64f08ea95e40b3fa208c2743b8eaba3296a63 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Mon, 27 Apr 2020 08:25:18 +0200
Subject: [PATCH] Suppress too_slow on test_parse_hypothesis

The generator is apparently too slow on busy systems, so suppress
the health check to avoid a test failure.
---
 tests/test_parse.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/test_parse.py b/tests/test_parse.py
index 6d58379..43494e3 100644
--- a/tests/test_parse.py
+++ b/tests/test_parse.py
@@ -2,13 +2,14 @@
 from __future__ import absolute_import, division, print_function
 
 import pytest
-from hypothesis import assume, given
+from hypothesis import assume, given, settings, HealthCheck
 
 from parver import ParseError, Version
 
 from .strategies import version_string, whitespace
 
 
+@settings(suppress_health_check=[HealthCheck.too_slow])
 @given(whitespace, version_string(), whitespace)
 def test_parse_hypothesis(prefix, version, suffix):
     Version.parse(prefix + version + suffix)
-- 
2.26.2

